frontend/public/components/utils/status-icon.tsx

```diff
@@ -22,7 +22,7 @@ export const StatusIcon: React.FunctionComponent<StatusIconProps> = ({status}) =
     case 'Running':
     case 'Updating':
     case 'Upgrading':
-      return <span className="co-icon-and-text"><Icon type="fa" name="refresh" className="co-icon-and-text__icon" /><CamelCaseWrap value={status} /></span>;
+      return <span className="co-icon-and-text co-icon-and-text--truncate"><Icon type="fa" name="refresh" className="co-icon-and-text__icon" /><CamelCaseWrap value={status} /></span>;
```

Comment:
We shouldn't keep `CamelCaseWrap` if we use `white-space: nowrap;` also. Does it make sense to still allow wrapping, but still have `overflow: hidden`?

If we're going to do this, I don't want to apply it globally for every case of `StatusIcon`, only where we need it.

All of the statuses below would need this as well.

Reply:
Correct, doesn't make sense to use `CamelCaseWrap` if it uses `co-truncate`. 

But since `co-truncate` needs to be a sibling of `co-icon-and-text`, how we would target specific page/locations if they all use the `status-icon.tsx` shared component, which uses `CamelCaseWrap`. 

Example markup needed
- for pods list page (would not use `CamelCaseWrap`)
```
<span class="co-icon-and-text co-truncate">
 <span aria-hidden="true" class="pf error-circle-o co-icon-and-text__icon"></span>ContainerCannotRun
</span>
```

- for pods details page (would use `CamelCaseWrap`)
```
<span class="co-icon-and-text">
 <span aria-hidden="true" class="pf error-circle-o co-icon-and-text__icon"></span>ContainerCannotRun
</span>
```